mods/default/crafting.lua

```diff
@@ -779,6 +779,15 @@ minetest.register_craft({
 	}
 })
 
+minetest.register_craft({
+	output = "default:large_cactus_seedling",
+	recipe = {
+		{"", "default:cactus", ""},
+		{"default:cactus", "default:cactus", "default:cactus"},
+		{"", "default:cactus", ""},
```

Comment:
Maybe a bit too expensive?

Reply:
Originally it was 2 cactus nodes, but that results in a very large number of seedlings per cactus. This way you get 2 from a large cactus which seems right, it also visually looks like the large cactus.